src/VisualStudio/Core/Def/Implementation/LanguageClient/AlwaysActiveLanguageClientEventListener.cs

```diff
@@ -47,13 +49,26 @@ internal class AlwaysActiveLanguageClientEventListener : IEventListener<object>
         /// </summary>
         public void StartListening(Workspace workspace, object serviceOpt)
         {
-            var token = this._asynchronousOperationListener.BeginAsyncOperation("LoadAsync");
             // Trigger a fire and forget request to the VS LSP client to load our ILanguageClient.
-            // This needs to be done with .Forget() as the LoadAsync (VS LSP client) synchronously stores the result task of OnLoadedAsync.
-            // The synchronous execution happens under the sln load threaded wait dialog, so user actions cannot be made in between triggering LoadAsync and storing the result task from OnLoadedAsync.
```

Comment:
FYI @gundermanc - this bit is no longer happening synchronously so unless the implementation has changed with the LSP client refactor, there could be cases where ctrl+Q search returns no results if it's called before the OnLoadedAsync task is stored.

We can't do this on the UI thread so ctrl+Q may need to be updated to be resilient where possible

Reply:
@gundermanc if we need that scenario to work, and for features like ctrl-q to be able to wait for servers to start up, then we'll need some mechanism for the client to asynchronously call in and wait for the server to be totally loaded.